pages/services.tsx

```diff
@@ -82,7 +82,11 @@ const Services = () => {
                 </h3>
               </Fade>
               {/* <Numbers /> */}
-              <Stats />
+              <Stats
+                followers={
+                  props.data?.data.data.user.edge_followed_by.count || 3720
```

Comment:
add optional chaining `?` to all so that this doesn't break.

Reply:
done